server/liveblog/themes/themes_assets/default/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "liveblog-default-theme",
-  "version": "3.6.68",
+  "version": "3.6.77",
```

Comment:
I think once this functionality is fully ready we should bump this to be a minor version, like `3.7.0`, it's ok for now though